internal/controllers/cryostat_controller.go

```diff
@@ -659,6 +676,66 @@ func (r *CryostatReconciler) deleteConsoleLink(ctx context.Context, cr *operator
 	return nil
 }
 
+func (r *CryostatReconciler) addCorsAllowedOriginIfNotPresent(allowedOrigin string, cr *operatorv1beta1.Cryostat) error {
+	reqLogger := r.Log.WithValues("Request.Namespace", cr.Namespace, "Request.Name", cr.Name)
+	apiServer := &configv1.APIServer{}
+	err := r.Client.Get(context.Background(), types.NamespacedName{Name: resources.ApiServerName}, apiServer)
+	if err != nil {
+		reqLogger.Error(err, "Failed to get APIServer config")
+		return err
+	}
+
+	allowedOriginAsRegex := regexp.QuoteMeta(allowedOrigin)
+
+	for _, origin := range apiServer.Spec.AdditionalCORSAllowedOrigins {
+		if origin == allowedOriginAsRegex {
+			return nil
+		}
+	}
+
+	apiServer.Spec.AdditionalCORSAllowedOrigins = append(
+		apiServer.Spec.AdditionalCORSAllowedOrigins,
+		allowedOriginAsRegex,
+	)
+
+	err = r.Client.Update(context.Background(), apiServer)
```

Comment:
I just noticed this one minor thing. Could you pass the `context.Context` to this method instead of using `context.Background()` for the update? The context parameter should always be first to make golint happy: https://github.com/golang/go/wiki/CodeReviewComments#contexts.